packages/ui/src/machines/authenticator/actors/signIn.ts

```diff
@@ -549,6 +554,16 @@ export function signInActor({ services }: SignInMachineOptions) {
         shouldSetupTOTP: (_, event): boolean => {
           return isExpectedChallengeName(getChallengeName(event), 'MFA_SETUP');
         },
+        shouldSelectMfaType: (_, event): boolean => {
+          console.error(
+            `ERROR: 'SELECT_MFA_TYPE' is not supported by Amplify UI yet.
+            Please use only one MFA type or set up the MFA selector using this document: https://docs.amplify.aws/lib/auth/mfa/q/platform/js/#advanced-use-cases/`
+          );
+          return isExpectedChallengeName(
+            getChallengeName(event),
+            'SELECT_MFA_TYPE'
```

Comment:
IF we don't support `SELECT_MFA_TYPE` this will always return `false` right?

Reply:
We don't know until this line